### ISSUE TITLE
Support thickness has to be less than chip_thickness in detector models

### ISSUE THREAD

**Issue by simonspa** [simonspa/allpix-squared#55]
In gitlab by @smaqbool on Jul 11, 2017, 18:55

Having made the detector model for h35demo, I encountered a strange bug which didn't allow me to specify the support thickness greater than the chip_thickness. The detector configuration file is pasted below:

```
type = "hybrid"

number_of_pixels = 80 300
pixel_size = 250um 50um

sensor_thickness = 100um
sensor_excess_left = 4250um
sensor_excess_right = 4250um

bump_sphere_radius = 3.0um
bump_cylinder_radius = 2.0um
bump_height = 3.0um
bump_offset = 0 0

chip_thickness = 760um

[support]
thickness = 750um
size = 53mm 94mm
offset = 0 0
location = "sensor"
#material = "plexiglass"

[support]
thickness = 50um
size = 53mm 94mm
offset = 0 0
location = "sensor"
material = "copper"
```

If I try to specify the 1st support thickness to greater than 760um, the program throws an exception. The actual thickness is supposed to be 1.4mm and I am unable to specify that. I get the following error when I try to do that:

> |18:48:37.642|  (STATUS) Running event 1 of 1000
> -------- EEEE ------- G4Exception-START -------- EEEE -------
> *** G4Exception : GeomMgt0002
>       issued by : G4SmartVoxelHeader::BuildNodes()
> PANIC! - Overlapping daughter with mother volume.
>          Daughter physical volume support_detector4_phys_1
>          is entirely outside mother logical volume wrapper_detector4_log !!
> *** Fatal Exception *** core dump ***
> -------- EEEE -------- G4Exception-END --------- EEEE -------
> 
> 
> *** G4Exception: Aborting execution ***
> Aborted (core dumped)

Additionally, I was unable to specify the material for the first support to either FR4 or plastic. I'll appreciate your comments.

Salamn

**Comment by simonspa** [comment]
In gitlab by @simonspa on Jul 17, 2017, 13:12

closed via commit 4a1372f58b1f9f863d4fc5276dbef13b127ca82d

**Comment by simonspa** [comment]
In gitlab by @simonspa on Jul 18, 2017, 10:25

mentioned in commit kwolters/allpix-squared@4a1372f58b1f9f863d4fc5276dbef13b127ca82d
